### PR TITLE
Add LTS kernel support

### PR DESCRIPTION
**Pull request by dingdang66686**

### CODE REVIEW

**Comment by dingdang66686** [comment]
Is this able to merge?

**Comment by Locietta** [comment]
Oops, forget to submit my review...

File: .github/workflows/build-lts.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        echo "CURRENT_VERSION=$CURRENT_VERSION" | tee -a $GITHUB_ENV
+        echo "RELEASED_VERSION=$RELEASED_VERSION" | tee -a $GITHUB_ENV
+
+        if [[ $CURRENT_VERSION != $RELEASED_VERSION || \
+              ($CURRENT_VERSION = $RELEASED_VERSION && 1 -gt "$RELEASED_MINOR") || \
+              "${{ github.event_name }}" = 'pull_request' ]]; then
+          echo "REBUILD_FLAG=1" | tee -a $GITHUB_ENV
+        else 
+          echo "REBUILD_FLAG=" | tee -a $GITHUB_ENV
+        fi
```
[the remark this answers is not in the document]
Comment (dingdang66686) [reply .github/workflows/build-lts.yaml:74]: Can we use RELEASE_TAG to identify different versions? I.E. Add "lts" version string in LTS release.

**Comment by Locietta** [comment]
I personally perfer not to touch the version string, how about something like `curl -sL -H 'Authorization: Bearer ${{ secrets.CUSTOM_GITHUB_TOKEN }}' https://api.github.com/repos/Locietta/xanmod-kernel-WSL2/tags\?per_page\=60 | jq -r '.[].name' | grep ${{ env.repo_branch }} | head -n 1`?

**Comment by dingdang66686** [comment]
It seems fine.

**Review by Locietta — APPROVED** [review]

**Comment by Locietta** [comment]
Everything looks in good shape now, thanks for your efforts!

However, I can't merge it right away util I fix downstream [scoop updater](https://github.com/Locietta/sniffer/blob/ad0191d2bef80b9d30f715d270df1de9e525d628/bucket/xanmod-WSL2.json#L34-L44) . It's another story though, and takes some extra time. But I will get this PR merged as soon as I can!

**Pull request merged**